CMakeLists.txt

```diff
@@ -115,7 +116,7 @@ cmake_policy(SET CMP0097 NEW)
 include(ExternalProject)
 ExternalProject_Add(datasketches
   GIT_REPOSITORY https://github.com/apache/datasketches-cpp.git
-  GIT_TAG 5.0.0
+  GIT_TAG master
```

Comment:
why?

Reply:
Oh, good catch. KLL needed the fix in 5.0.1 so need to flip it back to that.

Reply:
Fixed